package.json

```diff
@@ -16,7 +16,7 @@
     "@rjsf/core": "2.4.0",
     "@rjsf/material-ui": "2.4.0",
     "autosuggest-highlight": "3.1.1",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
```

Comment:
Please keep `package.json` unmodified unless necessary (e.g., if you are adding a new dependency) -- please keep this pinned to this version.